apps/test/integration/levelSolutions/studio/k1_6.js

```diff
@@ -0,0 +1,138 @@
+var testUtils = require('../../../util/testUtils');
```

Comment:
ahhh, I see; resurrecting a skipped test. Do we know why that test was marked as skip?